corehq/apps/domain/forms.py

```diff
@@ -169,9 +169,13 @@ class SnapshotSettingsForm(SnapshotSettingsMixin):
         help_text=ugettext_noop("An optional image to show other users your logo or what your app looks like"))
     video = CharField(label=ugettext_noop("Youtube Video"), required=False,
         help_text=ugettext_noop("An optional youtube clip to tell users about your app. Please copy and paste a URL to a youtube video"))
+    documentation_file = forms.FileField(label=ugettext_noop("Documentation File"), required=False,
+        help_text=ugettext_noop("An optional file to tell users more about your app."))
     cda_confirmed = BooleanField(required=False, label=ugettext_noop("Content Distribution Agreement"))
 
     def __init__(self, *args, **kw):
+        self.dom = kw.pop("domain", None)
```

Comment:
Is there a reason why you shortened "domain" to "dom"? When I see "dom" I think Document Object Model.